examples/vision/supervised-contrastive-learning.py

```diff
@@ -0,0 +1,186 @@
+"""
+Title: Supervised Contrastive Learning
+Author: [Khalid Salama](https://www.linkedin.com/in/khalid-salama-24403144/)
+Date created: 2020/11/01
+Last modified: 2020/11/01
+Description: Using supervised contrastive learning for image classification.
+"""
+
+## Introduction
+
+[Supervised Contrastive Learning](https://arxiv.org/abs/2004.11362) (Prannay Khosla et al.) is a training methodology that outperforms cross-entropy on supervised learning tasks. 
```

Comment:
Please keep text short (helps with code reviews)

Reply:
Could you suggest a shorter text for the intro?

Reply:
Sorry, I meant to say "keep text lines short" (hence the comment about code review). Please shorten all lines to be no longer than ~80-90 characters.

Reply:
done